### PR TITLE
DOC: Registration initialization and center of rotation docs.

### PR DESCRIPTION
**Pull request by zivy**
Adding prose on registration initialization and the setting of center
of rotation to the read-the-docs documents.

### CODE REVIEW

File: docs/source/registrationOverview.rst [hunk 1]
```diff
@@ -21,6 +21,32 @@ Code illustrating various aspects of the registration framework can be found in
...
+rotation angle, the farther you are from the fulcrum the larger the displacement. For numerical stability we do not want our computations
+to be sensitive to very small variations in the rotation angle, thus the ideal center of rotation is the point which minimizes the
+distance to the farthest point in our ROI:
+
+.. math::
+
+   p_{center} = \underset{p_{rotation}} {\arg\min}\ dist(p_{rotation}, \{p_{roi}\})
+
+
+Without additional knowlege we can only assume that the ROI is the whole fixed image. If your ROI is only in a sub
```
Comment (dave3d) [inline docs/source/registrationOverview.rst:47]: knowledge

**Review by dave3d — APPROVED** [review]
Looks good

File: docs/source/registrationOverview.rst (hunk 1 above)
Comment (zivy) [reply docs/source/registrationOverview.rst]: corrected.

**Review by blowekamp — COMMENTED** [review]
nice, thank you

**Pull request merged**